application-ckeditor-plugins/src/main/resources/xwiki-image/imageSelector.js

```diff
@@ -167,4 +122,10 @@ define('imageSelector', ['jquery', 'modal', 'resource', 'l10n!imageSelector'],
         });
       }
     });
+    return {
+      createModal: createModal,
```

Comment:
```suggestion
      open: createModal,
```

The goal of this API is not to create a modal but to **open** the image selector.